content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -58,22 +58,26 @@ kubectl apply -f https://kubernetes.io/examples/controllers/frontend.yaml
 ```
 
 You can then get the current ReplicaSets deployed:
+
 ```shell
 kubectl get rs
 ```
 
 And see the frontend one you created:
+
 ```shell
 NAME       DESIRED   CURRENT   READY   AGE
 frontend   3         3         3       6s
 ```
 
-You can also check on the state of the replicaset:
+You can also check on the state of the ReplicaSet:
```

Comment:
@tengqm , I am wondering if the `docs/concepts/workloadscontrollers/replicaset.md` file was rebased?

Reply:
I've got no complaints about conflicts so far so I'd assume git is smart enough to do the merge.

Reply:
OK